quiz-template.html

```diff
@@ -1,148 +1,188 @@
-<html>
+<!DOCTYPE html>
+
+<!--
+Copyright 2015 sincerelypeggy
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
 
-<head> 
-<title> Quiz Title </title>
-<!-- This quiz is meant to be used for personality tests but can be adapted to suit other quiz needs. --> 
-
-<style>
-.questions{
-	font:bold 14px Arial;
-}
-
-.answers{
-	font:normal 12px Arial;
-}
-
-</style>
-
-<script>
-/* Global variables*/
-var totalQuestions = 3;
-var totalChoices = 3;
-
-/* Different choice point totals */
-var xTotal = 0;
-var yTotal = 0; 
-var zTotal = 0;
-
-/* Different choice attributes */
-var categoryName;
-var description;
-
-/* Hides the quiz page */
-function hideHome(){
-	var div = document.getElementById('home');
-	div.style.visibility = "hidden";
-	div.style.display = "none";
-}	
-
-/* Calculates which category a user fits in */
-function grade(){
-	for(var q=1; q<=totalQuestions;q++){ // all the questions 
-		var currentQuestionName = "question" + q;
-        var currentQuestions = document.getElementsByName(currentQuestionName);
-        for (var c = 0; c < currentQuestions.length; c++) {
-            var radio = currentQuestions[c];
-			if(radio.checked != true){
-                continue;
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<html>
+  <head> 
+    <title>Quiz Title</title>
+    <!-- This quiz is meant to be used for personality tests but can be adapted to suit other quiz needs. --> 
+
+    <meta charset="utf-8">

```

Comment:
Tells the browser what character encoding the file is written in. Required if not setting it using an HTTP header.